utils/formatter/messageStatusJsonFormatter.ts

```diff
@@ -0,0 +1,11 @@
+import { MessageFormatter } from "@pagopa/fp-ts-kafkajs/dist/lib/KafkaTypes";
+import { RetrievedMessageStatus } from "@pagopa/io-functions-commons/dist/src/models/message_status";
+
+export const toJsonMessageStatus = (
+  messageStatus: RetrievedMessageStatus
+): string => JSON.stringify(messageStatus);
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export const jsonMessageStatusFormatter = (): MessageFormatter<RetrievedMessageStatus> => messageStatus => ({
+  value: toJsonMessageStatus(messageStatus)
```

Comment:
Could be useful to add the CF as message 'key' value?

Reply:
Sure, but I don't know if it is used effectively, since the previous implementation (with output binding) does not take care about key :)